liesel/model/nodes.py

```diff
@@ -1171,4 +1171,4 @@ def __getitem__(self, key) -> Var | Node:
         return self._nodes_and_vars[key]
 
     def __repr__(self) -> str:
-        return f"{type(self).__name__}<{self.name}>"
+        return f"{type(self).__name__}({self.name})"
```

Comment:
I think for the group, it would be nice to have the same style as for the Var, i.e. `Group(name="name")`